packages/firebase_ai/firebase_ai/example/analysis_options.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright 2021 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# in the LICENSE file.
+
+include: ../../../../analysis_options.yaml
+linter:
+  rules:
+    avoid_print: false
+    depend_on_referenced_packages: false
+    library_private_types_in_public_api: false
```

Comment:
If we have this, we can remove the firebase_ai/analysis_options.yaml

Reply:
can you be more specific? I'm trying to following the pattern in other plugins and might missed something.

Reply:
If it's the same as in other packages, let's keep that then